astronomy/trappist_dynamics_test.cpp

```diff
@@ -1176,29 +1180,49 @@ TEST_F(TrappistDynamicsTest, MathematicaPeriods) {
 }
 
 TEST_F(TrappistDynamicsTest, MathematicaTransits) {
-  Instant const a_century_later = system_.epoch() + 100 * JulianYear;
-  ephemeris_->Prolong(a_century_later);
-
-  TransitsByPlanet computations;
+  // Run this test with different ephemeris to make sure that the system is
+  // converged.
```

Comment:
Should this be a TEST_P instead?

Reply:
There would be a lot of boilerplate, so, no.